components/breadcrumb-item/package.json

```diff
@@ -0,0 +1,36 @@
+{
+	"name": "@vonage/vwc-breadcrumb-item",
+	"version": "2.26.0",
+	"description": "breadcrumb-item component",
+	"homepage": "https://github.com/Vonage/vivid/tree/master/components/breadcrumb-item#readme",
+	"license": "ISC",
+	"main": "vwc-breadcrumb-item.js",
+	"module": "vwc-breadcrumb-item.js",
+	"files": [
+		"*.js",
+		"*.ts",
+		"*.map"
+	],
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/vonage/vivid.git",
+		"directory": "components/breadcrumb-item"
+	},
+	"scripts": {
+		"build:typescript": "tsc -b",
+		"build:styles": "umbrella-style-modules",
+		"build": "yarn run build:styles && yarn run build:typescript"
+	},
+	"bugs": {
+		"url": "https://github.com/Vonage/vivid/issues"
+	},
+	"dependencies": {
+		"@vonage/vvd-foundation": "^2.26.0",
+		"lit-element": "^2.4.0",
```

Comment:
Missing core

Reply:
`vvd-core`?

Reply:
Yes it should be imported in all vivid `< 3` components

Reply:
Actually - it doesn't use any of these...
I've removed v-2 deps. Left only umbrella for build time dep.